scribble/app/views/comments/_comment.html.erb

```diff
@@ -0,0 +1,15 @@
+<p>
+  <strong>Name</strong>
+  <%= comment.commenter %>
+</p>
+
+<p>
+  <strong>Comment:</strong>
+  <%= comment.body %>
+</p>
+
+<p>
+  <%= link_to "Delete Comment", [comment.post, comment],
```

Comment:
Nice! I didn't know you could pass in an array of objects here to generate the nested 'show' path.